packit_service_fedmsg/consumer.py

```diff
@@ -78,15 +79,16 @@ def fedora_messaging_callback(self, message: Message):
         :param message: Message from Fedora message bus
         :return: None
         """
-        if message.body.get("user") != "packit":
-            logger.info("Not built by packit!")
-            return
-
         if message.topic not in INTERESTED_TOPICS:
             logger.debug("Not interested topic")
             return
 
-        logger.info(message.body.get("what"))
+        user = message.body.get("user") or message.body.get("owner")
```

Comment:
@csomh was recently [changing this](https://github.com/packit-service/packit-service-fedmsg/commit/cee751e74e4adbdd2b8557b213edf541f08ba08a) so I'd like to see his opinion here.

Reply:
This might even work :smiley:

I would still like to see the handling of Copr messages clearly separated from the Koji ones.

The reason: the two adhere to different schema, so although fields might have the same name (like "owner") they might not necessarily mean the same thing, or their meaning might diverge in the future. Mixing them together in an expression might lead to surprising results and it's also not obvious on first sight why these two fields are the ones checked and why in this order.

So being explicit about it now might save us some headaches during future debugging-sessions.

By explicit here I mean: first check what the message is about (Koji state change or Copr event) and then check the appropriate field to tell who started the build ("owner" in case of Koji, "user" in case of Copr) and act accordingly.

Reply:
> I would still like to see the handling of Copr messages clearly separated from the Koji ones.

- This feels like out-of-the-scope of this PR, I've only copied the existing workflow.
  - We handle them separately in the worker code. Here, we only need to get rid of foreign messages.
- I'll make the filtering more clear.

Reply:
@csomh PTAL, I've separated the checks.